lib/config.js

```diff
@@ -190,6 +190,7 @@ config.views.vars = {
   },
   minify: false,
   pageLayout: 'normal',
+  renderTimeout: 500,
```

Comment:
Add comment in line above that says `// maximum time to wait in milliseconds before assuming a page is fully rendered`